web/libs/datamanager/src/components/DataManager/Toolbar/instruments.js

```diff
@@ -21,37 +21,46 @@ const style = {
   justifyContent: 'space-between',
 };
 
-// Check if user is on trial
-const isTrialExpired = window.APP_SETTINGS.billing.checks.is_license_expired;
-// Check the subscription period end date
-const subscriptionPeriodEnd = window.APP_SETTINGS.subscription.current_period_end;
-// Check if user is self-serve
-const isSelfServe = isFF(FF_SELF_SERVE) && window.APP_SETTINGS.billing.enterprise;
-// Check if user is self-serve and has expired trial
-const isSelfServeExpiredTrial = isSelfServe && isTrialExpired && !subscriptionPeriodEnd;
-// Check if user is self-serve and has expired subscription
-const isSelfServeExpiredSubscription = isSelfServe && subscriptionPeriodEnd && new Date(subscriptionPeriodEnd) < new Date();
-// Check if user is self-serve and has expired trial or subscription
-const isSelfServeExpired = isSelfServeExpiredTrial || isSelfServeExpiredSubscription;
+/**
+ * Checks for Starter Cloud trial expiration.
+ * If expired it renders disabled Import button with a tooltip.
+ */
+const ImportButtonWithChecks = ({ size }) => {
+  const simpleButton = <ImportButton size={size}>Import</ImportButton>;
+  const isOpenSource = !window.APP_SETTINGS.billing;
+  // Check if user is self-serve; Enterprise flag === false is the main condition
+  const isSelfServe = isFF(FF_SELF_SERVE) && window.APP_SETTINGS.billing?.enterprise === false;
```

Comment:
```suggestion
  const isSelfServe = isFF(FF_SELF_SERVE) && !window.APP_SETTINGS.billing?.enterprise;
```

Reply:
here I explicitly check for `false` because `undefined` here is when billing is not set up. I check this condition separately, so even with `!` it will work, but semantic will be a bit different which I don't want

Reply:
This would mean OpenSource would be considered based on the loosened falsey check.